gazebo_plugins/include/gazebo_plugins/gazebo_ros_openni_kinect.h

```diff
@@ -115,7 +115,10 @@ namespace gazebo
     private: sensor_msgs::PointCloud2 point_cloud_msg_;
     private: sensor_msgs::Image depth_image_msg_;
 
+    /// \brief Minimum range of the point cloud
     private: double point_cloud_cutoff_;
```

Comment:
I'll merge this now. Feel free to submit new pr with renaming efforts, or file an issue to track for now. Thanks!